LICENCE

```diff
@@ -0,0 +1,26 @@
+Copyright (c) 2018, VSHN AG, info@vshn.ch
```

Comment:
The year is a bit outdated.

```suggestion
Copyright (c) 2021, VSHN AG, info@vshn.ch
```

Reply:
copy/paste from one of your other repos :see_no_evil: